### PR TITLE
Vue training - 01 -  Project setup and developer tools layout

### PR DESCRIPTION
**Pull request by rsuescun**
## Summary

-Front-End Bootstrap was added in `/vue-training` folder
-Layout of Developer Tools using Pug 

## Screenshots
screencapture-localhost-3000-2019-04-23-13_41_11

## Trello Card

https://trello.com/c/pME8GZHh/18-front-end-bootstrap

### CODE REVIEW

File: vue-training/src/scss/equalizer.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
+$equalizer-black: #000;
```
Comment (FrankIglesias) [inline vue-training/src/scss/equalizer.scss]: you can use [wolox-equalizer](https://www.npmjs.com/package/wolox-equalizer) package here

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: vue-training/src/scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+$tech-text-margin: 10px;
+
+.tech-box-list {
+  align-items: start;
+  align-self: center;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+  margin-bottom: 80px;
+  width: 1100px; //arbitrary value
```
Comment (FrankIglesias) [inline vue-training/src/scss/tech.scss]: use width and max-width properties here

File: vue-training/src/views/index/index.pug [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+          p.tech-box-text
+            | AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.
+        .tech-box
+          img.tech-box-logo(src='assets/tech/vue_icon.svg' alt='VueJS logo')
+          h3.tech-box-title VueJS
+          span.tech-box-launch Release: 2014
+          p.tech-box-text
+            | Vue.js (pronounced /vju&#x2D0;/, like view) is an open-source JavaScript framework for building user interfaces.
+        .tech-box
+          img.tech-box-logo(src='assets/tech/elm_icon.svg' alt='Elm logo')
```
Comment (FrankIglesias) [inline vue-training/src/views/index/index.pug]: remove logo alt

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: vue-training/src/views/index/index.pug (hunk 3 above)
Comment (rsuescun) [reply vue-training/src/views/index/index.pug]: Eliminated all attributes 'alt' but, why?

Comment (FrankIglesias) [reply vue-training/src/views/index/index.pug]: Only the logo word, as it's repeated it gives no additional info to the image

Comment (FrankIglesias) [reply vue-training/src/views/index/index.pug]: Do not delete the whole alt 😅

Comment (rsuescun) [reply vue-training/src/views/index/index.pug]: ups, sorry :stuck_out_tongue_winking_eye:

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**